website/docs/d/server.html.md

```diff
@@ -47,3 +47,4 @@ data "hcloud_server" "s_3" {
 - `placement_group_id` - (Optional, string) Placement Group ID the server is assigned to.
 - `delete_protection` - (bool) Whether delete protection is enabled.
 - `rebuild_protection` - (bool) Whether rebuild protection is enabled.
+- `shutdown_before_deletion` - (bool) Whether to try shutting the server down gracefully before deleting it.
```

Comment:
You added this argument to the datasource docs. It should go into the `Argument` section of the resource docs at `website/docs/r/server.html.md`

Reply:
Good catch, should be fixed now (added to both arguments and attributes)